AdyenActions/Actions/VoucherAction.swift

```diff
@@ -107,7 +128,7 @@ public class GenericVoucherAction: Decodable, AnyVoucherAction {
     public let merchantName: String
 
     /// The instruction url.
-    public let instructionsUrl: String
+    public let instructionsUrl: URL
```

Comment:
is this changing the public API? or is it okay in this case bc it is a `let` and the only way to create the object is via `init(from decoder: Decoder)`?

Reply:
yeah, if somebody access the `instructionsUrl ` they'll get a build error 🤔

Reply:
can we have a separate variable which is `URL` type? and use it for the voucher stuff. maybe we can put deprecated label on the current one, so we can remove it in 5.0

Reply:
deprecated `instructionsUrl` and created `instructionsURL` :-D